security/pkg/pki/util/keycertbundle.go

```diff
@@ -253,19 +253,18 @@ func (b *KeyCertBundle) UpdateVerifiedKeyCertBundleFromFile(certFile string, pri
 	return nil
 }
 
-// ExtractRootCertExpiryTimestamp returns the unix timestamp when the root becomes expires.
-func (b *KeyCertBundle) ExtractRootCertExpiryTimestamp() (float64, error) {
+// ExtractRootCertExpiryTimestamp returns when the root cert expires
+func (b *KeyCertBundle) ExtractRootCertExpiryTimestamp() (*time.Time, error) {
 	return extractCertExpiryTimestamp("root cert", b.GetRootCertPem())
 }
 
-// ExtractCACertExpiryTimestamp returns the unix timestamp when the cert chain becomes expires.
-func (b *KeyCertBundle) ExtractCACertExpiryTimestamp() (float64, error) {
+// ExtractCACertExpiryTimestamp returns when the cert chains expires
```

Comment:
Same comment as above

I will also include this in the release notes so that users who expected a UNIX timestamp to be returned can be aware